### ISSUE TITLE
macOS deployment target

### ISSUE THREAD

**Issue by dclelland** [kean/Nuke#155]
Hi there, I'm using Nuke with a project targeting macOS 10.10. There don't seem to be any 10.11 specific APIs in there - it seems to be compiling happily with the following post install hook:

```ruby
source 'https://github.com/CocoaPods/Specs.git'
platform :osx, '10.11'

use_frameworks!

pod 'Nuke', '~> 6.1'

post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == "Nuke"
            target.build_configurations.each do |config|
                config.build_settings["MACOSX_DEPLOYMENT_TARGET"] = "10.10"
            end
        end
    end
end
```

Would you object if I opened a PR to lower the deployment target?

**Comment by kean** [comment]
Hey! Yes, sure, I can definitely lower the deployment target. The one that I use now is almost arbitrary.

**Comment by dclelland** [comment]
Cool, I'll put in a PR today. Cheers!

**Comment by kean** [comment]
Pushed 6.1.1 with this fix. There was also disk cache path fix for macOS in 6.1 (#146). Please let me know if you find any issues or missing components on macOS. 😉
